### ISSUE TITLE
Too many elements have got the same color in base16 summerfruit dark

### ISSUE THREAD

**Issue by BonaBeavis** [vim-airline/vim-airline-themes#198]
This picture shows that, the middle section of the active status line, the inactive status line, the background of floating windows and the selected line in the location list have the same color.

![Screenshot_2019-12-12_22-12-40](https://user-images.githubusercontent.com/3330855/70751578-0607d100-1d31-11ea-81db-fb8859ac79c9.png)

I changed my theme by editing the summerfruit dark thme file and now it looks like this:
![Screenshot_2019-12-12_22-32-06](https://user-images.githubusercontent.com/3330855/70751589-0bfdb200-1d31-11ea-9a32-0f643ba29a8e.png)

I would like to create a pull request for updating the base16 templates. Against which repo should I do this?

**Comment by chrisbra** [comment]
you can use this repo. However please make sure to add a configuration switch and document the variable in the documentation. Thanks!

**Comment by chrisbra** [comment]
are you going to fix this here as well?

**Comment by BonaBeavis** [comment]
This fix was my first naive try. To really get a consistent look I had to modify [base16-vim: Base16 for Vim](https://github.com/chriskempson/base16-vim) too.

Additionally in his there are multiple base16 colorschemes for airline. First the meta base16 scheme with takes its'  highlights from the current base16 vim colorscheme and the other base16_* colorschmes rendered with these templates [fnurl/base16-vim-airline: Base16 for vim-airline](https://github.com/fnurl/base16-vim-airline). Two solutions for one problem.

I did not commit my changes, because the general feedback was that highlights are opinionated and my consistent look works for one of the base16 colorschemes and with my collection of vim plugins. I dont know about the other. 

![image](https://user-images.githubusercontent.com/3330855/73452256-cbeaa100-4369-11ea-9e4f-bbd8395c5f0e.png)

And I used a lighter grey for the `CursorLine` highlight I think this is highly likely a nonstarter for people who have set the cursorline.
